soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -11,3 +11,11 @@ config SOC_IT8XXX2
 	select ATOMIC_OPERATIONS_C
 
 endchoice
+
+config SOC_IT8XXX2_EXT_32K
+	bool "Use external 32.768 kHz clock source"
+	help
+	  Use an external 32.768 kHz clock source for PLL reference
+	  clock. A 32.768 kHz oscillator is required if set y.
+
+	  Default n to use the +/-2.3% internal clock generator.
```

Comment:
Seems like this should be a choice option instead of bool

i.e., one for internal, one for external

Reply:
Done.